bot/resources/tags/off-topic-names.md

```diff
@@ -0,0 +1,10 @@
+**Off-topic channels**
+
+There are three off-topic channels:
+• <#291284109232308226>
+• <#463035241142026251>
+• <#463035268514185226>
+
+The channel names change every night at midnight UTC and are fun meta references to jokes or conversations that happened on the server.
```

Comment:
```suggestion
The channel names change every night at midnight UTC and are often fun meta references to jokes or conversations that happened on the server.
```
Just a single suggestion, I don't really have an opinion on the tags themselves.